### ISSUE TITLE
mod conflict gets resolved in an ambiguous way

### ISSUE THREAD

**Issue by imre84** [minetest/minetest#13479]
##### Minetest version
```
5.7.0 (Linux)
```

##### OS / Hardware

Operating system:
CPU:

GPU model:
OpenGL version:

##### Summary

mod conflict gets resolved in an ambiguous way

##### Steps to reproduce

Hello,

I have this issue in 5.7.0 stable

```
<WARNING> 2023-05-03 10:54:40: [Main] Mod name conflict detected: "bones"
<WARNING> 2023-05-03 10:54:40: [Main] Will not load: .../bin/../games/minetest_game/mods/bones
<WARNING> 2023-05-03 10:54:40: [Main] Overridden by: .../worlds/testworld/worldmods/bones
<WARNING> 2023-05-03 10:54:40: [Main] Mod name conflict detected: "unified_inventory"
<WARNING> 2023-05-03 10:54:40: [Main] Will not load: .../worlds/testworld/worldmods/unified_inventory
<WARNING> 2023-05-03 10:54:40: [Main] Overridden by: .../bin/../mods/unified_inventory
```

maybe the issue is my understanding but I was expecting the world-specific mods to always override the more generic mods (which isn't the case as seen in the previous log snippet)

**Comment by fluxionary** [comment]
this has also confused me, but i'm not sure another ordering makes more sense. note that game and world mods are always enabled, but "global" mods are not.

**Comment by wsor4035** [comment]
labeled this as a bug rather than documentation, because while i could not find an explicit reference to order, looking https://github.com/minetest/minetest/blob/master/doc/lua_api.md#mod-load-path implies that the worldmods one should override the regular mods one, additionally, it makes the most sense.

unsubscribing

**Comment by rubenwardy** [comment]
I don't think this is a bug. Worldmods mods are automatically enabled. If you enable the  `/mods/` mod, it will override the `/worldmods/` mod. If you want to use the `/worldmods/` mod, just leave the `/mods/` mod disabled
